__init__.py

```diff
@@ -305,6 +311,27 @@ def handle_shopping_list_intent(self, message: Message) -> None:
         message.data["Entity"] = message.data.get("entity")
         self._handle_shopping_list(message)
 
+    def _handle_camera_image_actions(self, message: Message) -> None:
+        """Handler for camera image actions."""
+        entity = message.data["Entity"]
+
+        if not self.gui.connected:
+            self.speak_dialog('homeassistant.error.no_gui')
+            return
+
+        ha_entity = self._find_entity(entity, ['camera'])
+
+        if not ha_entity or not self._check_availability(ha_entity):
+            return
+
+        attributes = ha_entity['attributes']
+        entity_picture = attributes.get('entity_picture')
+
+        self.acknowledge()
+
+        self.gui.clear()
+        self.gui.show_image(f"{self.ha_client.url}{entity_picture}", override_idle=15)
```

Comment:
I'm assuming you've been testing this on an actual system. This just jumped out as a possible error point.

Does the client `url` or the `entity_picture` always have a `/` included?
I see in the test config the mock `entity_picture` has a `/` but it's not what I'd necessarily expect.

Reply:
`entity_picture` is a generated API URL to the image and is always an absolute path that starts always with a `/` and the URL should have no trailing slash, because it is generated using the IP/Domain and the Port: https://github.com/MycroftAI/skill-homeassistant/blob/20.08/ha_client.py#L88

So I think there is no problem.

Reply:
and welcome back @krisgesling :)

Reply:
Excellent - thanks for clarifying and for the warm return :smile: